app/src/main-process/menu/build-default-menu.ts

```diff
@@ -434,12 +434,12 @@ export function buildDefaultMenu({
     },
   ]
 
-  if (!hasCurrentPullRequest && enableStartingPullRequests()) {
```

Comment:
I originally had this `!hasCurrentPullReques` to follow the convention for the create pull request item which switches to an open pull request item in this scenario. But, I don't think there is a good reason to preview a user from seeing the pull request preview of an open PR.

Reply:
Wouldn't it be confusing? If there is already a PR, seeing a "Preview PR" could lead you to think there isn't one already 🤔

Reply:
Create pr would be disabled. This would just be for viewing the PR Diff which I found myself wanting to do after making some changes to existing PR. .. If you think it would be confusing, we could switch the verbage to "View Pull Request Diff" in this scenario. Another thought not necessarily for this PR, but the create button could change to a push button.